src/modules/scores-api.js

```diff
@@ -0,0 +1,51 @@
+const gameId = { name: 'Soccer Academy', id: 'ZCjCUXTffzaSkGhoabW0' };
+const baseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
+const scoresEndpoint = `/games/${gameId.id}/scores/`;
+
+const getData = async (url = '', method) => {
+  const response = await fetch(url, {
+    method,
+    mode: 'cors',
+    cache: 'no-cache',
+    credentials: 'same-origin',
+    redirect: 'follow',
+  });
+  return response.json();
+};
+
+const postData = async (url = '', data, method) => {
+  const response = await fetch(url, {
+    method,
+    mode: 'cors',
+    cache: 'no-cache',
+    credentials: 'same-origin',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    redirect: 'follow',
+    referrerPolicy: 'same-origin',
+    body: JSON.stringify(data),
+  });
+  return response.json();
+};
+
+const getScores = async () => {
+  try {
+    const data = await getData(baseUrl + scoresEndpoint, 'GET');
+    return { success: true, data };
+  } catch (error) {
+    return { success: false, error };
+  }
+};
+
+const postScores = async (obj) => {
+  try {
+    const data = await postData(baseUrl + scoresEndpoint, obj, 'POST');
+    return { success: true, data };
+  } catch (error) {
+    return { success: false, error };
+    // console.log(error);
```

Comment:
- [ ] it is bad to leave `console.log` in the production code. Console.log statements are used for debugging purposes and should be removed before the code is deployed to production. Leaving `console.log `statements in the production code can cause performance issues and can also expose sensitive information. so please remove it as the previous reviewer asked